src/adapters/http/client.ts

```diff
@@ -24,13 +24,13 @@ class HttpClientAdapter extends Adapter {
     const authConfig = await clientAuthConfig(this.serverName)
     const serverUrl = this.serverUrlExpanded
     for (const channelName of this.channelNames) {
-      const channelInfo = this.parsedAsyncAPI.channel(channelName)
-      const httpChannelBinding = channelInfo.binding('http')
-      const channelServers = channelInfo.servers()
+      const channelInfo = this.parsedAsyncAPI.channels().get(channelName)
```

Comment:
Have you tried this? I see the `url` variable on line 34 is using the `channelName` but in v3 the channel name is not the address as it used to be in v2. I don't think this code is working 🤔

Reply:
In this PR I am not changing the logic to parse to v3, I am just updating the parser to v2, glee still only supports the spec v2 documents.

Reply:
@fmvilas if you are talking about this 

```
        let url = `${serverUrl}/${channelName}`
```

I rechecked and this is giving me `ws://localhost:3000/price` as the output.

Reply:
If you're testing [this document](https://github.com/asyncapi/glee/blob/master/examples/crypto-websockets/client/asyncapi.yaml#L19), this is going to work because in v3 channelName === address. They're the same thing. But in v3, channelName and address can be different (and most probably will be different). For instance:

```yaml
asyncapi: 3.0.0
...
channels:
  thePrice:
    address: '/price'
    ...
```

In this case, `channelName` would be equal to `thePrice` but the `address` would be `/price`. So you don't want to chain with `channelName` to form the URL but, instead, you want to chain with the `address`. Does that make sense?